### PR TITLE
Test against AGP 8.3.0-alpha13

### PR DESCRIPTION
**Pull request by runningcode**

### CODE REVIEW

File: src/test/resources/expectedOutcomes/8.3.0-alpha13_outcomes.json [hunk 1]
```diff
@@ -35,10 +35,10 @@
     ":app:extractReleaseNativeSymbolTables"              :  "NO_SOURCE",
     ":app:extractReleaseVersionControlInfo"              :  "SUCCESS",
     ":app:generateDebugAssets"                           :  "UP_TO_DATE",
-    ":app:generateDebugResources"                        :  "UP_TO_DATE",
+    ":app:generateDebugResources"                        :  "FROM_CACHE",
```
Comment (runningcode) [inline src/test/resources/expectedOutcomes/8.3.0-alpha13_outcomes.json:38]: @cdsap do you think this is worth reporting to Google?

Comment (cdsap) [reply src/test/resources/expectedOutcomes/8.3.0-alpha13_outcomes.json]: yep, sounds good
let me check what's there in the task

**Review by cdsap — APPROVED** [review]

File: src/test/resources/expectedOutcomes/8.3.0-alpha13_outcomes.json (hunk 1 above)
Comment (cdsap) [reply src/test/resources/expectedOutcomes/8.3.0-alpha13_outcomes.json]: some findings, there is a change in the type of task used for `generate$BuildTypeResources`:
* In `8.3.0-alpha12` the type was DefaultTask, I assume registered during the plugin configuration: https://ge.solutions-team.gradle.com/s/quh4ztwynlohc/timeline?name=generateDebugResources 
* In `8.3.0-alpha14` the type is `ValidateResourcesTasks` https://ge.solutions-team.gradle.com/s/dy2sthbz4oqd2/timeline?name=generateDebugResources  More info about the [Task](https://cs.android.com/android-studio/platform/tools/base/+/mirror-goog-studio-main:build-system/gradle-core/src/main/java/com/android/build/gradle/internal/tasks/ValidateResourcesTask.kt;l=44?q=validatereso)

Comment (runningcode) [reply src/test/resources/expectedOutcomes/8.3.0-alpha13_outcomes.json]: Thanks for investigate. In that case, it seems this change is intentional.

**Pull request merged**